qiskit_experiments/visualization/plotters/base_plotter.py

```diff
@@ -464,6 +467,8 @@ def _default_figure_options(cls) -> Options:
             yval_unit=None,
             xval_unit_scale=True,
             yval_unit_scale=True,
+            xscale=None,
```

Comment:
Can you add option validator for valid option names? 
https://github.com/Qiskit/qiskit-terra/blob/163875e5d7729fe98cca16c46fc6b12408976964/qiskit/providers/options.py#L198-L205